vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -100,6 +110,34 @@ int main(int argc, char** argv)
         l_rc = l_vpdToolObj.readKeyword(l_vpdPath, l_recordName, l_keywordName,
                                         l_isHardwareOperation, l_filePath);
     }
+    else if (!l_dumpObjFlag->empty())
+    {
+        if (!l_objectOption->empty())
+        {
+            vpd::VpdTool l_vpdToolObj;
+
+            try
+            {
+                const nlohmann::json& l_sysConfigJson =
+                    vpd::jsonUtility::getParsedJson(INVENTORY_JSON_SYM_LINK);
+
+                nlohmann::json l_resultInJson = nlohmann::json::array({});
+                l_rc = l_vpdToolObj.dumpObject(l_vpdPath, l_sysConfigJson,
+                                               l_resultInJson);
+
+                if (vpd::constants::SUCCESS == l_rc)
+                {
+                    vpd::utils::printJson(l_resultInJson);
+                }
+            }
+            catch (std::runtime_error& l_ex)
+            {
+                std::cerr << "Object Dump failed due to error : " << l_ex.what()
+                          << std::endl;
+                return vpd::constants::FAILURE;
```

Comment:
should you just need to update l_rc in case if success, instead returning here, as return is there end if the function

Reply:
Done